src/Components/MyProfileFolder/MyProfile.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { useSelector } from 'react-redux'; // se usa para acceder al estado en el store
+import ProfileRocketItem from './ProfileRocketItem';
+import ProfileMissionItem from './ProfileMissionItem';
+
+const MyProfile = () => {
+  const rockets = useSelector((state) => state.Rockets);
+  const missions = useSelector((state) => state.Missions);
+  return (
+    <div className="container">
+      <div className="profilediv" id="flex">
+        <div className="joinedmissions">
+          <h1>My Missions</h1>
+
+          {missions.filter((item) => item.joined === true).length === 0 ? (
+            <h2>No Missions Joined</h2>)
+            : (
+              <ul className="missionlist" id="flex">
+                {missions.filter((item) => item.joined === true).map((mission) => {
```

Comment:
- [ ] **[OPTIONAL]:** A better approach to handle these two filters and improve readability would be to:

 Define two variables

```js
const joinedMissions = missions.filter((item) => item.joined === true)
const noJoinedMissions = joinedMissions.length === 0
...

```

Now, the checks would look like this:

```suggestion
          {noJoinedMissions ? (
            <h2>No Missions Joined</h2>)
            : (
              <ul className="missionlist" id="flex">
                {joinedMissions.map((mission) => { ...
```

You could apply the same improvement to the rockets section! 🤓 🔧